examples/AvroGeneric/AvroGeneric.csproj

```diff
@@ -1,71 +1,12 @@
-﻿<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+﻿<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This is nice, but is it safe with older .NET frameworks?

Reply:
yes, though probably not old versions of visual studio. but same constraints as other projects in the solution.

note: I'll do general verification of cross framework compatibility again when we do an RC.